Blog.API/Blog.API/Program.cs

```diff
@@ -3,12 +3,16 @@
 using Microsoft.AspNetCore.Builder;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
 using SpanJson.AspNetCore.Formatter;
 
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 
+builder.Logging.ClearProviders();
+builder.Logging.AddConsole();
```

Comment:
As I remember we already discussed that. These lines are doing nothing. We can get rid of them

Reply:
in my case, it would't work